packages/core/admin/admin/src/core/apis/CustomFields.js

```diff
@@ -1,5 +1,25 @@
 import invariant from 'invariant';
 
+const ALLOWED_TYPES = [
+  'string',
```

Comment:
I'm a bit picky here but can you sort the array alphabetically? (VSCODE has a feature for this: select multiple rows, command + shift + p => type sort line)

Reply:
BTW that's a good move it will make the CTB simpler as you won't have to handle specific use case for components and DZ!